docs/source/algorithms/CalculatePlaczekSelfScattering-v1.rst

```diff
@@ -0,0 +1,88 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+This algorithm takes an incident spectrum function and it's derivative and
+calculates the time-of-flight Placzek scattering correction from the workspaces
+material sample and detector info. [1]_ [2]_ [3]_ For obtaining the incident spectrum
+from a measurement (ie beam monitors or calibrant sample), the
+:ref:FitIncidentSpectrum <algm-FitIncidentSpectrum> can provide the necessary inputs.
+
+Usage
+-----
+
+**Example: calculate Placzek self scattering correction using a sample detector setup** [4]_
+
+.. code:: python
+
+    from mantid.simpleapi import *
+    import matplotlib.pyplot as plt
+    import numpy as np
+
+    # Create the workspace to hold the already corrected incident spectrum
+    incident_wksp_name = 'incident_spectrum_wksp'
+    binning_incident = "%s,%s,%s" % (0.1, 0.02, 5.0)
+    binning_for_calc = "%s,%s,%s" % (0.2, 0.02, 4.0)
+    binning_for_fit = "%s,%s,%s" % (0.15, 0.02, 4.5)
+    incident_wksp = CreateSampleWorkspace(
+        OutputWorkspace=incident_wksp_name,
+        NumBanks=10,
+        XMin=0.1,
+        XMax=5.0,
+        BinWidth=0.002,
+        Xunit='Wavelength')
+    incident_wksp = ConvertToPointData(InputWorkspace=incident_wksp)
+
+    # Spectrum function given in Milder et al. Eq (5)
+    def incidentSpectrum(wavelengths, phiMax, phiEpi, alpha, lambda1, lambda2,
+                         lamdaT):
+        deltaTerm = 1. / (1. + np.exp((wavelengths - lambda1) / lambda2))
+        term1 = phiMax * (
+            lambdaT**4. / wavelengths**5.) * np.exp(-(lambdaT / wavelengths)**2.)
+        term2 = phiEpi * deltaTerm / (wavelengths**(1 + 2 * alpha))
+        return term1 + term2
+
+    # Variables for polyethlyene moderator at 300K
+    phiMax = 6324
```

Comment:
You have a mix of camelCase and snake_case in here, please use a singly style (I'm not actually bothered which, our guidelines say camalCase, but the rest of the world says snake_case).